shub/auth.py

```diff
@@ -0,0 +1,48 @@
+import os
+import netrc
+
+from os.path import expanduser
+
+from click import ClickException
+
+OS_WIN = True if os.name == 'nt' else False
+NETRC_FILE = expanduser('~/_netrc') if OS_WIN else expanduser('~/.netrc')
+
+
+def find_api_key():
+    """
+    Raises:
+        ClickException: if no credentials are found
+    """
+    key = os.getenv("SHUB_APIKEY")
```

Comment:
Although we want to keep users in the login/logout workflow, I think $SHUB_APIKEY is convenient for automated scripts, right? It can be used to deploy projects from CI, for example.